composer/modules/web/source-gen/tree.g

```diff
@@ -199,7 +199,8 @@ Return
    ;
 
 Service
-   : <annotationAttachments>* <documentationAttachments>* <deprecatedAttachments>* service < <serviceTypeStruct.source> > <name.value> bind <boundEndpoints-joined-by,>* { <variables>* <resources>* }
+   : <isServiceTypeUnavailable?> <annotationAttachments>* <documentationAttachments>* <deprecatedAttachments>* service                                <name.value> bind <boundEndpoints-joined-by,>* { <variables>* <resources>* }
+   |                             <annotationAttachments>* <documentationAttachments>* <deprecatedAttachments>* service < <serviceTypeStruct.source> > <name.value> bind <boundEndpoints-joined-by,>* { <variables>* <resources>* }
```

Comment:
@NipunaMarcus If this specific rule is moved to the top we can avoid the isServicetypeUnavailable check.

Reply:
aa didn't know that .. i'll edit these with the new changes coming to source gen